src/Jira/Api.php

```diff
@@ -150,6 +153,24 @@ public function editIssue($issueKey, $params)
     }
 
 
+     /**
+     * Delete issue
+     *
+     * @param $issueKey should be YOURPROJ-221
+     * @param $deleteSubtasks if all subtask should be deleted
+     * @return mixed
+     */
+    public function deleteIssue($issueKey, $deleteSubtasks = 'true')
```

Comment:
The trick that JIRA needs boolean specified as strings is it's internal thing and doesn't need to be exposed via API client interface. This way please replace `'true'` with `true`.